README.md

```diff
@@ -185,29 +185,36 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Intel® Extension for Transformers](https://github.com/intel/intel-extension-for-transformers)  ![](https://img.shields.io/github/stars/intel/intel-extension-for-transformers.svg?style=social) - An Innovative Transformer-based Toolkit to Accelerate GenAI/LLM Everywhere.
 * [Inference](https://github.com/roboflow/inference) - A fast, production-ready inference server for computer vision supporting deployment of many popular model architectures and fine-tuned models. With Inference, you can deploy models such as YOLOv5, YOLOv8, CLIP, SAM, and CogVLM on your own hardware using Docker.
 * [Jina](https://github.com/jina-ai/jina)  ![](https://img.shields.io/github/stars/jina-ai/jina.svg?style=social) - Cloud native search framework that   supports to use deep learning/state of the art AI models for search.
-* [KServe](https://github.com/kserve/kserve) ![](https://img.shields.io/github/stars/kserve/kserve.svg?style=social) - Serverless framework to deploy and monitor machine learning models in Kubernetes - [(Video)](https://www.youtube.com/watch?v=hGIvlFADMhU).
+* [KsanaLLM](https://github.com/pcg-mlp/KsanaLLM) ![](https://img.shields.io/github/stars/pcg-mlp/KsanaLLM.svg?style=social) - Serverless framework to deploy and monitor machine learning models in Kubernetes - [(Video)](https://www.youtube.com/watch?v=hGIvlFADMhU).
+* [KServe](https://github.com/kserve/kserve) ![](https://img.shields.io/github/stars/kserve/kserve.svg?style=social) - KsanaLLM is a high performance and easy-to-use engine for LLM inference and serving.
 * [Langfuse](https://github.com/langfuse/langfuse) ![](https://img.shields.io/github/stars/langfuse/langfuse.svg?style=social) - Langfuse is an open source observability & analytics solution for LLM-based applications.
+* [Lepton AI](https://github.com/leptonai/leptonai) ![](https://img.shields.io/github/stars/leptonai/leptonai.svg?style=social) - LeptonAI Python library allows you to build an AI service from Python code with ease.
 * [LightLLM](https://github.com/ModelTC/lightllm) ![](https://img.shields.io/github/stars/ModelTC/lightllm.svg?style=social) - LightLLM is a Python-based LLM (Large Language Model) inference and serving framework, notable for its lightweight design, easy scalability, and high-speed performance.
 * [LLMonitor](https://github.com/LLMonitor/llmonitor) ![](https://img.shields.io/github/stars/LLMonitor/llmonitor.svg?style=social) - Observability & analytics for AI apps and agents.
 * [LocalAI](https://github.com/mudler/LocalAI) ![](https://img.shields.io/github/stars/mudler/LocalAI.svg?style=social) - LocalAI is a drop-in replacement REST API that's compatible with OpenAI API specifications for local inferencing.
+* [lunary](https://github.com/lunary-ai/lunary) ![](https://img.shields.io/github/stars/lunary-ai/lunary.svg?style=social) - Open-source observability & prompt platform for LLMs.
```

Comment:
Let's remove this LLM-prompt-management one to merge this PR, and discuss as part of our sync